### PR TITLE
Allow childprocess 1.x - issue #6907

### PR DESCRIPTION
**Pull request by twalpole**
- [ X ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

---
This change is [Reviewable](https://reviewable.io/reviews/seleniumhq/selenium/6948)

### CODE REVIEW

File: rb/Gemfile.lock [hunk 1]
```diff
@@ -17,7 +17,6 @@ GEM
       safe_yaml (~> 1.0.0)
     diff-lcs (1.3)
     ffi (1.9.25)
-    ffi (1.9.25-x64-mingw32)
```
Comment (p0deje) [inline rb/Gemfile.lock]: Can you please revert this change? It is added when installing gems on Windows so, once merged, it will be added back on the next `bundle install` run on Windows.

File: rb/Gemfile.lock [hunk 2]
```diff
@@ -71,4 +70,4 @@ DEPENDENCIES
   yard (~> 0.9.11)
 
 BUNDLED WITH
-   1.16.3
```
Comment (p0deje) [inline rb/Gemfile.lock]: Let's not bump bundler version since we ship 1.16.3 by default. Another approach would be to bump it in `third_party/rb'?

File: rb/Gemfile.lock (hunk 1 above)
Comment (twalpole) [reply rb/Gemfile.lock]: Ok -- is there a specific reason Gemfile.lock is even checked in? or possibly added and then gitignored if it being updated is a problem?

Comment (p0deje) [reply rb/Gemfile.lock]: Frankly, I don't remember why exactly it was added. Probably because we used cached gems and if there is no Gemfile - it pulls more recent versions. I haven't checked that, so maybe we're safe to remove it altogether?

Comment (luke-hill) [reply rb/Gemfile.lock]: :point_up: Just for reference, this is the reason. I remember chatting with simon about this a while ago when I did a minor tweak
